install/jakartaee/bin/ts.jte

```diff
@@ -1972,9 +1972,9 @@ ts_home=${ts.home}
 #######################################################################
 ## platform.mode is used by jpa tests to know whether to use
 ## the Java EE Programming Model for service and port access.
-## platform.mode=javaEE		Java EE Programming Model
+## platform.mode=jakartaEE		Java EE Programming Model
```

Comment:
Should we also change ```Java EE``` to ```Jakarta EE``` for this pr?

Reply:
> @alwin-joseph
> I didn't finish reviewing yet but am thinking that further (separate) changes should be separate commits.
Yes,  there are more such references to Java EE, I suggest those can go in  another PR if possible.
> 
> Also, I'm concerned about merging the change to Standalone TCKs before teams working on changing Standalone TCKs are done with their changes (someone will have to resolve conflicts.) Should we merge our changes last? 
I'm not sure that helps because some of our changes we want to merge before we consider a standalone TCK complete for ballot.

I think there would be conflicts mostly in https://github.com/eclipse-ee4j/jakartaee-tck/pull/415. But I suggest merging this change first as it would be easier to resolve conflicts in other PRs. I will take care of resolving conflicts occurring in other documentation PRs due to this change (assuming I would have the permissions to do that in other changes)